src/server.ts

```diff
@@ -34,6 +35,9 @@ app.use(bodyParser.urlencoded());
 app.use(cookieParser());
 app.use("/graphql", validateJwtMiddleware);
 
+// enable ssl redirect
+app.use(sslRedirect());
+
```

Comment:
whats that for ? Certificates ? Are we not supposed to do it on frontend though? I guess on both side maybe